content/blog/2021-11-23-mlops-architecture-playground.md

```diff
@@ -0,0 +1,118 @@
+date: 2021-11-23
+description: >
+  Introducing a new way the MLOps Open-Source Community can contribute to the
+  greater good.
+descriptionLong: >
+  Having an issue conceptualizing and communicating your MLOps workflows?  Let's
+  work together to solve the problems we all face in the MLOps Architecture
+  Playground.
+picture: 2021-11-02/playground.png
+pictureComment: Join Us on the MLOps Architecture Playground
+author: jeny_defigueiredo
+commentsUrl: https://discuss.dvc.org/t/blog-mlops-architecture-playground/934
+tags:
+  - MLOps
+  - Architecture
+  - Reproducibility
+  - DVC
+  - Workflows
+  - Community
+---
+
+## Intro
+
+As the Community Manager at Iterative, I am charged with taking care of and
+serving our Community . I scour the internet ingesting all the amazing content
+our users create, talk to you at Meetups, and have listened to your challenges
+and pain points in calls with you or through Discord. Over this time, some
+things I've noticed in the Community and life in general:
+
+- The best (and most shared) content has good images and diagrams to drive
+  clarity
+- Given the flexibility and hands-off-your-code characteristics of our tools, it
+  can sometimes be challenging for people to conceptualize their workflows
+- diagram images can often be hard to read in a blog post because they take up a
+  lot of space and are illegible when condensed
+- the best solutions and ideas most often come from many places
+- many hands make light work
+- we have an awesome Community of exceptional humans!
+
+So digesting these things and attending other community events where a Miro
+board was used to collaborate and solve problems, I came up with the idea for
+the **MLOps Architecture Playground** in Miro.
+
+![MLOps Architecture Playground by Iterative.ai](/uploads/images/2021-11-23/mlops-playground.png)
+_Join us in the MLOps Archtecture Playground!
+([Source link](https://www.ravirajag.dev/blog/mlops-summary))_
+
+### What?
+
+What this will be is a place for people in the greater MLOps Community to come
+and work out their architecture issues. This could be a small piece of the
+puzzle or the greater architecture of your entire workflow and tool stack.  
+What we aim to provide you is a place to work it out and all the components you
+need to create it fast (or as fast as possible, given that this is complicated
+stuff!). Your work in here will likely help other Community members struggling
+with the same issues.
+
+https://media.giphy.com/media/xUOxfecWsLzAYSUP6w/giphy.gif
+
+### Who is this board for?
+
+Everyone! You don't have to use only Iterative tools to use this board. We know
+that our tools are good building blocks in your MLOps tools stack, but there are
+many other tools. And our tools integrate with many. Feel free to work out your
+diagram even if it does not use our tools. We're all in this together!
+
+### How will this work?
+
+1. Use this link [needed from MailChimp, figuring out], to enter the MLOps
```

Comment:
Do I need Miro account?

Reply:
No you don't need a Miro account to join and can still edit the board.  Just the person who hosts needs a Miro account. ✅